arch/aarch64/cpu.c

```diff
@@ -1,5 +1,11 @@
 #include <kernel/cpu.h>
 
+//////////////////////////////////////////////////////////////////////////
```

Comment:
Can you please use the less verbose Doxygen syntax that's already used elsewhere? See for example, the file `arch/x86_64/include/arch/interrupts.h` for an example. Also, if there no parameters or return value, please just omit the Doxygen tag.

Reply:
Yup.

Reply:
Yup.